resources/assets/js/components/Radio.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export interface RadioProps {
+  /** HTML id of the input element */
+  id: string;
+  /** HTML name of the input element */
+  name: string;
+  /** Holds text for label associated with input element */
+  label: string;
+  /** boolean indicating if this radio is selected */
+  checked?: boolean;
+  /** The value of the input */
+  value?: string | number | string[];
+  /** Event listener which fires when a change event occurs (varies on input type) */
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Event listener which fires when a input loses focus */
+  onBlur?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const Radio: React.FunctionComponent<RadioProps> = ({
+  id,
+  name,
+  label,
+  checked,
+  value,
+  onBlur,
+  onChange,
+}): React.ReactElement => (
+  <label htmlFor={id}>
```

Comment:
If the label element wraps the input, I don't believe the htmlFor field is necessary.

Reply:
Resolved in 233d7dc